spec/lib/healthchecks/queue_dead_set_check_spec.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+require 'sidekiq/testing'
+require 'okcomputer'
+require 'healthchecks'
+require 'scholarsphere/redis_config'
+
+RSpec.describe HealthChecks::QueueDeadSetCheck do
+  before do
+    Sidekiq::Testing.disable!
+    redis_config = Scholarsphere::RedisConfig.new
+    Sidekiq.configure_server do |config|
+      config.redis = redis_config.to_hash
+    end
+
+    Sidekiq.configure_client do |config|
+      config.redis = redis_config.to_hash
+    end
+
+    ds = Sidekiq::DeadSet.new
+    ds.clear
```

Comment:
I think this could be replaced with:

``` ruby
Sidekiq::Testing.disable!
Sidekiq::DeadSet.new.clear
```

Reply:
Yeah, i do this a lot. i think in blocks, and it ends up in my text editor in that way